api/manifest.go

```diff
@@ -145,11 +145,17 @@ func downloadManifest(deploymentRequest NaisDeploymentRequest) (naisManifest Nai
 
 	// not provided, using defaults
 	urls := createManifestUrl(deploymentRequest.Application, deploymentRequest.Version)
+	var lastError error
 	for _, url := range urls {
 		manifest, err := fetchManifest(url)
 		if err == nil {
 			return manifest, nil
 		}
+		lastError = err
+	}
+
+	if lastError != nil {
+		return NaisManifest{}, fmt.Errorf("unable to fetch manifest: %v\n", lastError.Error())
 	}
 
 	return NaisManifest{}, fmt.Errorf("No manifest found on the URLs %s, or the url %s\n", urls, manifestUrl)
```

Comment:
Vil ikke denne endringen bety at vi aldri når denne linjen? Er vel bedre å bake feilen inn i denne feilmeldingen?

Reply:
viss `urls` mot formodning er tom, så vil en treffe default return, men enig at kanskje man burde refaktorert litt.

Reply:
Ah, sant. Men ja, beste med en liten refaktorering kanskje?

Reply:
+1 for å refaktorere. Kanskje kan https://godoc.org/github.com/pkg/errors være til hjelp?